### ISSUE TITLE
In-app bug reporting

### ISSUE THREAD

**Issue by gino-m** [google/ground-android#30]

**Comment by shobhitagarwal1612** [comment]
Can we close this in favor of current crash/error reporting?

**Comment by gino-m** [comment]
This was actually referring to user feedback. Updated description to reflect this.

**Comment by shobhitagarwal1612** [comment]
@gino-m Can we revisit this to allow users to report errors manually via email or some forum?

**Comment by gino-m** [comment]
> @gino-m Can we revisit this to allow users to report errors manually via email or some forum?

I think this is useful, but before doing this we should fix all crashes in Crashlytics (#crashl  and ensure changes to the survey don't break data collection ([google/ground-platform/iisues/](https://github.com/google/ground-platform/issues/1701). Would you like to handle the Android side of that change?

**Comment by shobhitagarwal1612** [comment]
IIUC, we need to mark the `job`, `task`, and `multiple choice` local entities as deleted and hide them from UI, right?

**Comment by gino-m** [comment]
> IIUC, we need to mark the `job`, `task`, and `multiple choice` local entities as deleted and hide them from UI, right?

Yep! Moving discussion on that to https://github.com/google/ground-platform/issues/1701 for now.
